src/Jobs/Recruit.Vacancies.Jobs/VacancyStatus/LiveVacancyStatusInspector.cs

```diff
@@ -22,16 +21,20 @@ public LiveVacancyStatusInspector(ILogger<LiveVacancyStatusInspector> logger, IJ
 
         internal async Task InspectAsync()
         {
-            var vacancies = await _client.GetLiveVacancies();
-
+            var vacancies = (await _client.GetLiveVacancies()).ToList();
+            int numberClosed = 0;
+            
             foreach (var vacancy in vacancies)
```

Comment:
`foreach (var vacancy in vacancies.Where(v => v.ClosingDate <= _timeProvider.Now))` instead of the if condition being called in the loop maybe.

Reply:
I don't mind this foreach. There may be other processing we want to do in future for other live vacancies

Reply:
👍